src/Commands/InstallCommand.php

```diff
@@ -3,22 +3,277 @@
 namespace Rapidez\Core\Commands;
 
 use Illuminate\Console\Command;
+use Illuminate\Support\Collection;
+
+use function Laravel\Prompts\confirm;
+use function Laravel\Prompts\multiselect;
+use function Laravel\Prompts\pause;
+use function Laravel\Prompts\select;
 
 class InstallCommand extends Command
 {
-    protected $signature = 'rapidez:install';
+    protected $signature = 'rapidez:install {--frontendonly : Just copy over the frontend files}';
 
     protected $description = 'Install Rapidez';
 
+    protected Collection $selectedPackages;
+
+    protected bool $dockerInstall = false;
+
     public function handle()
     {
-        copy(__DIR__ . '/../../package.json', base_path('package.json'));
-        copy(__DIR__ . '/../../postcss.config.js', base_path('postcss.config.js'));
-        copy(__DIR__ . '/../../tailwind.config.js', base_path('tailwind.config.js'));
-        copy(__DIR__ . '/../../vite.config.js', base_path('vite.config.js'));
-        copy(__DIR__ . '/../../yarn.lock', base_path('yarn.lock'));
-        copy(__DIR__ . '/../../.prettierrc.js', base_path('.prettierrc.js'));
-
-        $this->info('Done 🚀');
+        if ($this->option('frontendonly')) {
+            $this->frontend(true);
+
+            return;
+        }
+
+        $this
+            ->intro()
+            ->magento()
+            ->validate()
+            ->frontend()
+            ->packages()
+            ->finalize();
+    }
+
+    protected function intro()
+    {
+        $this->info(str_repeat('=', 37));
+        $this->info('  ____             _     _          ');
+        $this->info(" |  _ \ __ _ _ __ (_) __| | ___ ____");
+        $this->info(" | |_) / _` | '_ \| |/ _` |/ _ \_  /");
+        $this->info(' |  _ < (_| | |_) | | (_| |  __// / ');
+        $this->info(" |_| \_\__,_| .__/|_|\__,_|\___/___|");
+        $this->info('            |_|                     ');
+        $this->info('  Welcome to the Rapidez installer!');
+        $this->info(str_repeat('=', 37));
+        $this->newLine();
+        $this->warn('> You will get some questions, if you do not know what to choose;');
+        $this->warn('> Just enter through, the defaults are a good start.');
+        $this->newLine(2);
+
+        return $this;
+    }
+
+    protected function magento()
+    {
+        if (! confirm('Do you have a Magento installation up-and-running?')) {
```

Comment:
We're going 3 blocks deep here, invert these if statements?

```php
if (confirm('Do you have a Magento installation up-and-running?')) {
    return $this;
}

if (!confirm(
    label: 'Shall we setup a demo Magento installation for you in Docker?',
    hint: 'This will run `docker-compose up -d` and takes a while, make sure Docker is running!'
)) {
    $this->error('Please setup a Magento installation first!');
    exit;
}

[...]
```